src/hotspot/share/opto/compile.hpp

```diff
@@ -827,18 +835,39 @@ class Compile : public Phase {
 
   const CompilationFailureInfo* first_failure_details() const { return _first_failure_details; }
 
+  bool failing(DEBUG_ONLY(bool no_stress_bailout = false)) {
+    if (failing_internal()) {
+      return true;
+    }
+#ifdef ASSERT
+    // Disable stress code for PhaseIdealLoop verification
```

Comment:
Can you expand the comment here and add the reason why? From a comment above, you mentioned that it is not easy to make it work. I guess it's fine to just mention that here.